CONTRIBUTING.md

```diff
@@ -550,6 +550,21 @@ pull request. This PR is not signed and has the label `user-signing-required`. T
 done by a maintainer of the repository. Check out the PR and push an amending commit into
 the existing branch. Then, the PR can be merged.
 
+For an example, take a look at this pull request: [Bundle dependabot pull requests](https://github.com/weaveworks-liquidmetal/flintlock/pull/655).
+
+It was created by [this](https://github.com/weaveworks-liquidmetal/flintlock/actions/runs/4335326798) action triggered on the main branch.
+
+Once the pull request was opened, a maintainer created a commit into it by checking out the branch, then running a `git commit --amend`.
+
+```bash
+# git pull if not already up-to-date
+git checkout bundler-1678007734
+git commit --amend
+git push --set-upstream origin bundler-1678007734 -f
```

Comment:
We can drop the `--set-upstream origin` With checkout, it's already set.

Reply:
That's true, yah. It was only needed when you checkout the Pr.